taskfiles/utils-cmake.yaml

```diff
@@ -88,3 +99,66 @@ tasks:
         {{- range .EXTRA_ARGS}}
         "{{.}}"
         {{- end}}
+
+  # Downloads a CMake project tar file from `URL` and then generates, builds, and installs the
+  # project. We purposely omit `sources` and `generates` as we defer to `cmake` to decide whether it
+  # should perform any actions. However, the download and extraction will be skipped if unnecessary.
+  #
+  # @param {string} NAME Project name used for directory names.
```

Comment:
nit: if you change this to `DEP` or `DEP_NAME` it will keep the required args alphabetical with `DEP*` being the first arg.
But this is really unnecessary

Reply:
Technically, the cmake project doesn't need to be a dependency so I think it is better to leave it as `NAME`.